content/release-notes/2022-07-19.md

```diff
@@ -4,10 +4,10 @@ label: 'Storage'
 
 ### What's new
 
-- Safekeeper: Added support for backing up WAL to S3 for disaster recovery.
-- Safekeeper: Added support for downloading WAL from S3 on demand.
-- Safekeeper: Switched to etcd subscriptions to keep pageservers up to date regarding safekeeper status.
-- Safekeeper: Implemented JWT authentication in the Safekeeper HTTP API.
+- Safekeeper: Added support for backing up Write-ahead Logs (WAL) to S3 storage for disaster recovery.
+- Safekeeper: Added support for downloading Write-ahead Logs (WAL) from S3 storage on demand.
+- Safekeeper: Switched to etcd subscriptions to keep Pageservers up to date regarding Safekeeper status. etcd is a key-value store for critical distributed system data.
```

Comment:
Personally, not a fun when a sentence starts with lovercase, so I'd avoid putting `etcd` in the beginning of it

Also in markdown maybe we can use backtick formatting -- `etcd` instead of just etcd

Reply:
I turned etcd into a link to the etcd site, which should address both issues here :)